tests/behaviour/expectations/behaviour.ts

```diff
@@ -3691,5 +3691,51 @@ export const expectations = flatten(
         ]),
       ]),
     ]),
+    new Dir('benchmark', [
+      new File(
+        'BaseJumpRateModelV2',
+        'BaseJumpRateModelV2',
+        ['10000000', '0', '11000000', '0', '10000000', '0', '10', '0', '0'],
+        [
+          Expect.Simple(
+            'getSupplyRate',
+            ['500', '0', '100', '0', '200', '0', '2', '0'],
+            ['0', '0'],
+          ),
+          Expect.Simple(
+            'utilizationRate',
+            ['500', '0', '100', '0', '200', '0'],
+            ['250000000000000000', '0'],
+          ),
+          Expect.Simple(
+            'updateJumpRateModel',
+            ['10000000', '0', '11000000', '0', '10000000', '0', '7', '0'],
+            [],
+          ),
+        ],
+      ),
+      File.Simple('ERC20_storage', [
+        Expect.Simple('deposit', ['74', '0', '500', '0'], []),
+        Expect.Simple('transferFrom', ['74', '0', '68', '0', '400', '0', '74', '0'], ['1']),
+        Expect.Simple('withdraw', ['80', '0', '74', '0'], []),
+        Expect.Simple('get_balance', ['74', '0'], ['20', '0']),
+      ]),
+      new File(
+        'ERC20',
+        'ERC20',
+        ['0', '0'],
```

Comment:
Please give these the name and ticker from the blog:https://medium.com/nethermind-eth/warp-2-0-transpiling-directly-from-solidity-to-cairo-9bf41a6d26ee. You should be able to use the same inputs stated there.